.github/actions/project-management-action/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# Container image that runs your code
```

Comment:
All this github-docker actions with dtq images, links you not need. It is our mistake we created the PR with this. If it will be OK we would like to keep them in this PR and do not remove them from the `dtq-dev` branch because we need that pipelines. And later we will create another PR where this pipelines will be removed, can we agree like this? 
Thank you.